### ISSUE TITLE
Auto Select Coins on Connection

### ISSUE THREAD

**Issue by Sh4r3catyt** [UnamSanctam/UnamWebPanel#148]
Hey, can you maybe add to the Panel after a Bot is connected what coins are the best for GPU VRAM? Size?

**Comment by UnamSanctam** [comment]
Well the web panel doesn't know how much VRAM the GPU has, and even then there isn't really a "best" coin, it all depends on preference and they all have their own pros and cons. For example, many mine ETC on 8GB cards because it uses less resources (VRAM in this case) so it will impact the user experience less and thus cause less user detections. And mining the new support Ubiq could technically be the most worth since it only uses 1.6GB of VRAM, while it's less profitable overall it could in practice be the most profitable to mine due to longevity and being able to mine on nearly all GPUs (though older ones will have lower hashrate obviously).

**Comment by Sh4r3catyt** [comment]
Yes but i basically mean for example if the gpu has 8gb it will mining ETH that you can setup it in a config if gpu has 4gb then mining ETC and 6gb+ = ETH and for other coins that you can setup for the gpu ram size
